Client/src/components/maze.tsx

```diff
@@ -0,0 +1,93 @@
+import { useEffect, useState } from "react";
+import '../css/maze.css';
+import { Graph, value } from "../utils/graph";
+import { generateMaze } from "../utils/maze";
+import { minionType } from "../utils/types";
+import MazeTile from "./mazeTile";
+import Minion from "./minion";
+
+function Maze({boxSize, setMazeCompleted, setCurrentMinion, minions, setCurrentTile, currentGraph, setCurrentGraph, height, width}: {
+  boxSize:number, 
+  height: number,
```

Comment:
Task with creation of a Type as DOD. I'm thinking about having an external class Maze to externalize all this actions

Reply:
What does DOD stand for?

Reply:
Definition Of Done